src/openms/source/KERNEL/MSExperiment.cpp

```diff
@@ -460,13 +460,18 @@ namespace OpenMS
     bool meta_present = false;
     for (Size i = 0; i < spectra_.size(); ++i)
     {
-      if (spectra_[i].getFloatDataArrays().size() != 0 || spectra_[i].getIntegerDataArrays().size() != 0 || spectra_[i].getStringDataArrays().size() != 0)
+      if (spectra_[i].getFloatDataArrays().size() != 0 
+        || spectra_[i].getIntegerDataArrays().size() != 0 
+        || spectra_[i].getStringDataArrays().size() != 0)
       {
         meta_present = true;
       }
```

Comment:
@timosachsenberg TODO: why not for chromatograms?

Reply:
Yes I think this is missing